docker/docker.go

```diff
@@ -25,6 +26,14 @@ const (
 )
 
 func main() {
+
+	for j := 1; j < len(os.Args); j++ {
+		if os.Args[j] == "-d" || os.Args[j] == "--daemon" {
+			log.Println("-d is deprecated. Please use daemon command")
+			os.Args[j] = "daemon"
+		}
+	}
+
```

Comment:
Can you elaborate ? All we are doing is replacing '-d/--daemon' with 'daemon' keyword. So that docker daemon is started using `sudo docker daemon` rather than `sudo docker -d`